ext/fg/js/popup-proxy-host.js

```diff
@@ -116,7 +116,8 @@ class PopupProxyHost {
 
     async _onApiContainsPoint({id, x, y}) {
         const popup = this._getPopup(id);
-        return await popup.containsPoint(x, y);
+        const rootPagePoint = PopupProxyHost._convertPopupPointToRootPagePoint(popup, x, y);
```

Comment:
For consistency with line 156, this should probably just destructure to `[x, y]`. The spread is going to do so anyways, so I don't think it saves anything.

Reply:
I somehow thought I had to keep the variable names the same, but that's not required because `popup.containsPoint` doesn't take an object that has to be restructured from the existing names. I'll destructure it to `rootX` and `rootY` and pass them on to the function.

Reply:
Oh, I didn't know that you can also destructure without assignment!

Reply:
You can do `[x, y] = getArray();` without needing `let` or `const`. Technically you can do `[x, y] = [y, x]` to swap. The syntax is a bit more strange when destructing an object, because you can't do inline objects due to being ambiguous with block scopes, so you have to do `({x, y}) = getObject();`

Reply:
Makes sense, I think that's why I had the idea that you couldn't do what I did in https://github.com/FooSoft/yomichan/pull/395/commits/9fef0751f3dc7f213f158a50124371f0c6fb5a17.